packages/global-styles/src/components/global-styles-variations/index.tsx

```diff
@@ -176,20 +192,20 @@ const GlobalStylesVariations = ( {
 							showOnlyHoverView={ showOnlyHoverViewDefaultVariation }
 							onSelect={ () => onSelect( baseGlobalStyles ) }
 						/>
-						{ ! splitPremiumVariations && premiumStyles }
+						{ ! splitPremiumVariations && customStyles }
 					</div>
 				</div>
 				{ splitPremiumVariations && (
 					<div className="global-styles-variations__type">
 						<div className="global-styles-variations__header">
 							<h2>
-								{ translate( 'Premium style', 'Premium styles', {
-									count: premiumStyles.length,
+								{ translate( 'Custom style', 'Custom styles', {
+									count: customStyles.length,
 								} ) }
 							</h2>
-							<p>{ premiumStylesDescription }</p>
+							<p>{ description ?? premiumStylesDescription }</p>
 						</div>
-						<div className="global-styles-variations">{ premiumStyles }</div>
+						<div className="global-styles-variations">{ customStyles }</div>
```

Comment:
I think this is part of a different issue (https://github.com/Automattic/dotcom-forge/issues/2857) which is solved by a different PR (https://github.com/Automattic/wp-calypso/pull/78848)

Reply:
I got confused by having this issue highlight these texts then: https://github.com/Automattic/dotcom-forge/issues/2858

Deans PR should get merged before this one, so there shouldn't be any issues.